### PR TITLE
[Feature-1] First feature has been implemented

### PR DESCRIPTION
**Pull request by prad-j**
Following changes has been made:

- new-feature.txt file added
- Feature-1 folder has been added

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature-1 branch
```
Comment (pramod-dj) [inline feature1/one-more-file.txt:1]: This is good change and the application is working as per requirement.
Great 
P

Comment (prad-j) [reply feature1/one-more-file.txt:1]: Thanks Pramod for your review.

**Review by pramod-dj — APPROVED** [review]
Great job Prad

**Comment by pramod-dj** [comment]
Good

**Review by pramod-dj — APPROVED** [review]
Approved

**Review by prad-j — COMMENTED** [review]
OK

**Review by pramod-dj — APPROVED** [review]
OK done

**Review by pramod-dj — APPROVED** [review]
OK great

**Comment by prad-j** [comment]
Very good.
It's now good to go

**Comment by prad-j** [comment]
Reopened the pull request after closing it by mistake

**Pull request merged**